include/deal.II/base/bounding_box.h

```diff
@@ -416,6 +445,21 @@ namespace internal
 #ifndef DOXYGEN
 
 
+template <int spacedim, typename Number>
+inline BoundingBox<spacedim, Number>::BoundingBox(
+  const BoundingBox<spacedim, Number> &other)
+  : boundary_points(other.boundary_points)
+{}
```

Comment:
Can't you use the `= default` copy constructor, since you don't do anything special here?